climada/entity/exposures/litpop/gpw_population.py

```diff
@@ -112,49 +115,39 @@ def get_gpw_file_path(gpw_version, reference_year, data_dir=SYSTEM_DIR, verbatim
     """
+    if data_dir is None:
+        data_dir = SYSTEM_DIR
+
     # get years available in GPW data from CONFIG and convert to array:
-    years_available = np.array([year.int() for year in \
-                                CONFIG.exposures.litpop.gpw_population.years_available.list()
-                               ])
+    years_available = np.array([
+        year.int() for year in CONFIG.exposures.litpop.gpw_population.years_available.list()
+    ])
+
     # find closest year to reference_year with data available:
     year = years_available[np.abs(years_available - reference_year).argmin()]
-    if verbatim and (year != reference_year):
+    if verbose and year != reference_year:
         LOGGER.warning('Reference year: %i. Using nearest available year for GPW data: %i',
-                    reference_year, year)
+                       reference_year, year)
 
-    # check if file is available for given GPW version,
+    # check if file is available for given GPW version, construct GPW file path from CONFIG:
     # if available, return full path to file:
-    file_path = data_dir / \
-        (CONFIG.exposures.litpop.gpw_population.filename_gpw.str() % (gpw_version, year))
-    if file_path.is_file():
-        if verbatim:
-            LOGGER.info('GPW Version v4.%2i', gpw_version)
-        return file_path
-    # try to construct GPW file path from CONFIG:
-    file_path = data_dir / \
-        (CONFIG.exposures.litpop.gpw_population.dirname_gpw.str() % (gpw_version, year)) / \
-        (CONFIG.exposures.litpop.gpw_population.filename_gpw.str() % (gpw_version, year))
-    if file_path.is_file():
-        if verbatim:
-            LOGGER.info('GPW Version v4.%2i', gpw_version)
-        return file_path
-    # if no input file was found, FileExistsError is raised
-    if SYSTEM_DIR.joinpath('GPW_help.pdf').is_file():
-        subprocess.Popen([str(SYSTEM_DIR.joinpath('GPW_help.pdf'))], shell=True)
-        raise FileExistsError(f'The file {file_path} could not '
-                              + 'be found. Please download the file '
-                              + 'first or choose a different folder. '
-                              + 'Instructions on how to download the '
-                              + 'file has been openend in your PDF '
-                              + 'viewer.')
-    raise FileExistsError(f'The file {file_path} could not '
-                          + 'be found. Please download the file '
-                          + 'first or choose a different folder. '
-                          + 'The data can be downloaded from '
-                          + 'http://sedac.ciesin.columbia.edu/'
-                          + 'data/collection/gpw-v4/sets/browse, '
-                          + 'e.g., https://sedac.ciesin.columbia.edu/downloads/data/'
-                          + f'gpw-v4/gpw-v4-population-count-rev{gpw_version}/'
-                          + f'gpw-v4-population-count-rev{gpw_version}_2020_30_sec_tif.zip '
```

Comment:
This is the main reason for this PR: Before this PR, the download URL in the error message would always point to the 2020 data of GPW instead of the year that is actually missing.